app/assets/javascripts/discourse/app/components/modal/bulk-topic-actions.gjs

```diff
@@ -289,13 +318,18 @@ export default class BulkTopicActions extends Component {
           @isLoading={{this.loading}}
           @title={{i18n "topics.bulk.performing"}}
         >
-          <div>
+          <div class="topic-bulk-actions-modal__selection-info">
             {{htmlSafe
               (i18n
                 "topics.bulk.selected"
                 count=@model.bulkSelectHelper.selected.length
               )
             }}
+
+            {{#if this.showSoleCategoryTip}}
+              {{i18n "topics.bulk.all_categories_same"}}
+              {{htmlSafe this.soleCategoryBadgeHTML}}
+            {{/if}}
```

Comment:
A nit with copy, the following is topic-count safe and ignores the presence or absence of restricted tags (would be weird to mention restricted tags if there are none)

<img width="668" alt="Screenshot 2024-04-11 at 4 30 36 PM" src="https://github.com/discourse/discourse/assets/1555215/843afa63-ea2f-4b9a-a5fd-f03477722d8c">

Reply:
I think there is a need because then you know it's the fact that you're only choosing topics from one category that unlocks special powers.

I see what you mean about the restricted tags wording though...let's just go with what you have here "You have selected 1 topic from category: X"